svgnative/src/SVGStringParser.cpp

```diff
@@ -936,25 +954,37 @@ static bool ParseColor(CharIt& pos, const CharIt& end, ColorImpl& paint, bool su
             std::int32_t r{};
             std::int32_t g{};
             std::int32_t b{};
+            bool r_percentage = false;
```

Comment:
The rest of the code uses camel case. Also, nit, use intialization lists.
```cpp
bool rPercentage{};
```

Reply:
Oh, another nit, boolean values should be descriptive:

`hasRPercentage` or `isRPrercentage`... depending on the context.

Reply:
```cpp
bool hasRPercentage{};
bool hasGPercentage{};
bool hasBPercentage{};
```